src/modules/dashboard/components/ActionsPage/ActionsComponents/MintTokenMotion.tsx

```diff
@@ -134,6 +140,28 @@ const MintTokenMotion = ({
   };
   const motionStyles = MOTION_TAG_MAP[motionState || MotionState.Invalid];
 
+  const getStakeShownSide = useCallback((): StakeSide => {
+    if (selectedStakeSide) {
+      return selectedStakeSide;
+    }
+
+    if (bigNumberify(motionNAYStake || 0).eq(0)) {
+      return StakeSide.Motion;
+    }
+
+    return StakeSide.Both;
+  }, [selectedStakeSide, motionNAYStake]);
+  const previousTotalStakeStep = useCallback(
+    () => setSelectedStakeSide(null),
+    [],
+  );
+
+  const stakeShownSide = getStakeShownSide();
+  const isOnStakingPhase =
```

Comment:
I would suggest `isStakingPhase` - just a bit simpler with already lots of things to understand in the code.